CUDA Kernels: Use per-operator headers (4/4)

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #71217
* #71216
* __->__ #71215
* #71214
* #71213

Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

peterbell10 committed:
CUDA Kernels: Use per-operator headers (4/4)

Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949902).

peterbell10 committed:
Rebase on master on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Update on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949902).

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "CUDA Kernels: Use per-operator headers (4/4)"


Splitting this into multiple PRs to keep the diffs more managable.

Differential Revision: [D33949902](https://our.internmc.facebook.com/intern/diff/D33949902)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33949902).
